### ISSUE TITLE
[Kim'jael Indeed!] (Azshara) bugged with FFA loot in party.

### ISSUE THREAD

**Issue by OMAOMC** [elysium-project/bug-tracker#693]
Cannot properly loot the required items for quest when in a party with Free For All loot.

**Comment by OMAOMC** [comment]
OK, so I did some further research and I found out what was causing the issue.
The quest doesn't appear to be bugged, and can be completed without issue.

HOWEVER: If you are doing the quest in a party and/or in a party with FFA loot activated - a lot of issues start to arise and the quest items do not drop properly from the boxes.

I managed to complete the quest, by dropping the quest, (important)destroying all the items I had acquired for the quest, picking up the quest again and then doing it alone WITHOUT A PARTY.

And it worked just fine.
There seems to be an issue with the loot tables when in party.

Either the party is causing the bug somehow, but much more likely was the Free For All loot that was activated when I was doing it in a party.
